.vscode/tasks.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Update cr/crd files",
+            "command": "olm/./update-cr-crd-files.sh",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Build and push custom che-operator image: '${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/che-operator:nightly'",
+            "command": "docker build -t ${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/che-operator:nightly . && docker push ${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/che-operator:nightly",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Port-forward che-server debug",
+            "command": "kubectl patch checluster eclipse-che -n che --type=merge -p '{\"spec\":{\"server\":{\"cheDebug\": \"true\"}}}' || true && chePod=$(kubectl get pods -n che -o yaml | yq -r \".items[] | select(.metadata.name | startswith(\\\"che\\\") and (startswith(\\\"che-operator\\\") | not)) | .metadata.name\") && echo ${chePod} | kubectl port-forward --namespace che pod/${chePod} 5005:8000",
```

Comment:
Do we really need this? 
We have `chectl server:start --debug` to start che-server in debug mode
and `chectl server:debug` for port forwarding.